scripts/new_project.kts

```diff
@@ -238,9 +270,21 @@ object NewProject {
         }
     }
 
-    private fun executeCommand(command: String) {
+    /**
+     * Execute a shell command
+     *
+     * Runtime.getRuntime().exec(String) will partition a command automatically, based on white spaces.
+     * If a file path contains a white space, it will not be able to find the file and result in an error.
+     *
+     * -> Example: "Desktop/My Projects/android-templates" -> ["Desktop/My", "Projects/android-templates"]
+     * -> Reference: https://stackoverflow.com/questions/33077129/java-runtime-exec-with-white-spaces-on-path-name
+     *
+     * @param [command]: A command partioned in to an array of strings
+     */
+    private fun executeCommand(command: Array<String>) {
```

Comment:
How about setting the argument as a `vararg`? We might be able to shorten the calls to this method by not creating an array explicitly, what do you think? 🤔 
```suggestion
    private fun executeCommand(vararg command: String) {
```
And calls could look like
```kotlin
executeCommand(
    "sh",
    "$projectPath${fileSeparator}gradlew",
    "-p",
    "$projectPath",
    "clean"
)